File: MSKU_java/Labworks/lab1/primefinder/PrimeFinder.java
class primefinder{
	public static void main(String[] args){
		String decleration = " is a prime number.";
		for(int number=1; number<100; number ++){
			int x=0;
			for(int divider=1; divider<=number; divider ++){
				if (number%divider==0){
					x++;
				}
			}
			if(x==2){
				System.out.println(number+decleration);
			}

		}

	}
}

File: MSKU_python/misc/primefinder.py
# ##Hi Guys, Lets find some primes...
import time

# Main idea of this code is, when all the code runs top to down, ultimate value of "x" will determine if "number" is prime or not.
# ##dfdg


def primefinder(number):
    # # Be aware that divider starts at 1. Becasuse we assume the number itself is already counted.
    divider = 1
    list1 = []

    # Be aware that stop condition of for loop is half of the number. It is a
    # very fundamental rule of dividing.
    for i in range(1, int((number / 2) + 1), 1):

        if number % i == 0:
            divider = divider + 1
            list1.append(i)

        if divider > 2:
            print(list1, "[...]")
            print(number, "is NOT a prime number.\n")
            break

    if divider == 2:
        list1.append(number)  # Number itself is added to the list manually.
        print(list1)
        print(number, "is a prime number.\n")


y = 34568970897654324567
while True:

    primefinder(y)
    y = y + 1
